Source/Core/DolphinQt/CMakeLists.txt

```diff
@@ -1,15 +1,24 @@
-if (NOT Qt5_DIR AND MSVC)
+if (Qt6_DIR)
```

Comment:
For compatibility reasons since it's relatively common to use ``Qt5_DIR`` / ``Qt6_DIR`` to specify what Qt version to used (and we ourselves used the former in the past)